pandas/tests/io/excel/test_writers.py

```diff
@@ -838,6 +838,26 @@ def test_to_excel_multiindex_no_write_index(self, path):
         # Test that it is the same as the initial frame.
         tm.assert_frame_equal(frame1, frame3)
 
+    def test_to_excel_empty_multiindex(self, path):
+        # Test writing and re-reading an empty MI. GH 19543.
+
+        # Initial non-MI frame.
+        frame1 = DataFrame([[], [], []]).T
+
+        # Add a MI.
+        frame2 = frame1.set_index([0, 1])
```

Comment:
Could you define both of them explicitly without using any other functions like .T or set_index?

Reply:
Also please call them df, result and expected